litmus-portal/frontend/README.md

```diff
@@ -75,115 +50,68 @@ frontend
+2. Available Scripts
 
 In the project directory, you can run:
 
-#### 🐱‍💻 **Run dev server**
-
-```
+🐱‍💻 Run dev server
 npm start
-```
-
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-The page will reload if you make edits.<br>
+Runs the app in the development mode.
+Open http://localhost:3000 to view it in the browser. The page will reload if you make edits.
 You will also see any lint errors in the console.
-<br><br>
 
-#### 🔴 **Run unit tests**
 
-```
+🔴 Run unit tests
 npm test
-```
-
-Runs unit tests in cypress. You can do a `npx cypress open` to launch this in interactive mode in the browser.
-Cypress records a video for each spec file when running tests during `cypress run`. Videos are not automatically recorded during `cypress open`.
-Video recording can be turned off entirely by setting `video` to `false` from within your configuration.
+Runs unit tests in cypress. You can do a npx cypress open to launch this in interactive mode in the browser. Cypress records a video for each spec file when running tests during cypress run. Videos are not automatically recorded during cypress open. Video recording can be turned off entirely by setting video to false from within your configuration.
 
-```
 npm run unit:ci
-```
 
-<br>
-
-#### 🛠️ **Build the project**
-
-```
+🛠️ Build the project
 npm run build
-```
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-The build is minified and the filenames include the hashes.
+Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance. The build is minified and the filenames include the hashes.
 
-<br>
 
-#### 🏴🏴‍☠️ **Lint your code**
-
-```
+🏴🏴‍☠️ Lint your code
 npm run lint
-```
-
-ESLint statically analyzes your code to quickly find problems.<br>
-You would be able to see any lint errors in the console based on the rules set on `.eslintrc`
+ESLint statically analyzes your code to quickly find problems.
+You would be able to see any lint errors in the console based on the rules set on .eslintrc
 
-<br>
 
-#### ♻️ **Format your Code**
-
-```
+♻️ Format your Code
 npm run format
-```
-
-Formats your document according to the prettier guidelines and then does a `eslint --fix`
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-`Prettier` for formatting and `linters` for catching bugs! 🐛
-<br><br>
+Formats your document according to the prettier guidelines and then does a eslint --fix Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance. Prettier for formatting and linters for catching bugs! 🐛
 
-### `3. Tech Stack`
 
-<br>
+3. Tech Stack
```

Comment:
This is also supposed to be a bold list item which is linked with the Table of Contents.